keras_cv/models/object_detection_3d/center_pillar.py

```diff
@@ -140,18 +35,19 @@ class MultiHeadCenterPillar(keras.Model):
     regression heads on the feature map.
 
     Args:
-      backbone: the backbone to apply to voxelized features.
-      voxel_net: the voxel_net that takes point cloud feature and convert
-        to voxelized features.
-      multiclass_head: a multi class head which returns a dict of heatmap
-        prediction and regression prediction per class.
-      label_encoder: a LabelEncoder that takes point cloud xyz and point cloud
-        features and returns a multi class labels which is a dict of heatmap,
-        box location and top_k heatmap index per class.
-      prediction_decoder: a multi class heatmap prediction decoder that returns
-        a dict of decoded boxes, box class, and box confidence score per class.
-
-
+        backbone: the backbone to apply to voxelized features.
+        voxel_net: the voxel_net that takes point cloud feature and convert
+            to voxelized features. KerasCV offers a `DynamicVoxelization` layer
+            in `keras_cv.layers.object_detection_3d.voxelization` which is a
+            reasonable default for most detection use cases.
+        multiclass_head: a multi class head which returns a dict of heatmap
+            prediction and regression prediction per class.
+        label_encoder: a LabelEncoder that takes point cloud xyz and point cloud
```

Comment:
Is `LabelEncoder` a class to inherit?

Reply:
Whoops, this shouldn't be here anymore because this arg doesn't exist.